libs/platform/src/lib/components/button/button.component.html

```diff
@@ -1,14 +1,6 @@
-<button
-    fd-button
-    [attr.id]="id"
-    [attr.title]="title"
-    [attr.aria-label]="title"
-    [compact]="compact"
-    [glyph]="glyph"
-    [disabled]="disabled"
-    (click)="onBtnClick($event)"
-    [fdType]="type"
-    [ngStyle]="{ width: width }"
->
+<button #fdButton fd-button [attr.id]="id" [attr.title]="title" [attr.name]="name" [attr.aria-label]="title"
+    [attr.type]="type" [attr.value]="value" [attr.aria-selected]="ariaSelected" [attr.aria-disabled]="ariaDisabled"
```

Comment:
is there an issue with the formatter? why is it wrapping it up?

Reply:
Done by Prettifier.

Reply:
when i use the same piece of code in my vscode, prettier retains the older formatting. I guess you can still save without formatting, it looks more readable in the previous formatting. Right click -> Command Palette -> Save without formatting